PWGHF/TableProducer/HFTrackIndexSkimsCreator.cxx

```diff
@@ -421,15 +423,25 @@ struct HfTagSelTracks {
 
       iDebugCut = 4;
       if (doCutQuality.value && (debug || statusProng > 0)) { // FIXME to make a more complete selection e.g track.flags() & o2::aod::track::TPCrefit && track.flags() & o2::aod::track::GoldenChi2 &&
-        UChar_t clustermap = track.itsClusterMap();
-        if (!(track.tpcNClsFound() >= tpcNClsFound.value && // is this the number of TPC clusters? It should not be used
-              track.flags() & o2::aod::track::ITSrefit &&
-              (TESTBIT(clustermap, 0) || TESTBIT(clustermap, 1)))) {
+        bool hasGoodQuality = true;
+        if (useIsGlobalTrack) {
+          if (track.isGlobalTrack() != (uint8_t) true) {
```

Comment:
Why in case of global tracks you don't require a hit in one of the two innermost layers of the ITS? Shouldn't we maybe disentangle the two selections to be able to apply either one of the two or both? Thanks!

Reply:
Ciao Fabrizio,
according to what presented by Mario in his AW talk, the isGlobalTrack requirement should already contain a hit, actually I also checked and there's indeed a call selectedTracks.SetRequireHitsInITSLayers(1, {0, 1});
Anyway, in general at some point we could find out that the isGlobalTrack selection could be not optimal for HF: in that case I think the best solution would be to have a dedicated status flag, as isGlobalTrack but checking a generic-HF selection, I discussed this with Mattia and Mario, and it would be possible to have it.

Reply:
Ciao @fcolamar, I missed that the requirement of a hit in the first two layers was already included in `isGlobalTrack`, thanks for the explanation!